skipruntime-ts/native/src/Runtime.sk

```diff
@@ -22,14 +22,22 @@ class ExistingSubscriptionException() extends Exception {
   }
 }
 
-class Params(value: Map<String, PValue>) extends SKStore.File uses Orderable {
+class Params(
```

Comment:
To disable unsafe construction:
```suggestion
class Params private (
```

Reply:
I pulled this branch and checked that adding `private` here does seem to work. Did you not like that idea, or this comment got missed, or?

Reply:
I made the change but ended up switching to a different representation in #603 which didn't need the private modifier.  But I'll add it here and then land so that reviewing that is easier, per your comment over there!